test/user.test.js

```diff
@@ -0,0 +1,74 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+import app from '../app';
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+
+describe('Register Route: /api/v1/users/register', () => {
```

Comment:
You're almost there. Wrap all these `describe` blocks in a single `describe` block. It means you'd end up having one large describe and children describe blocks.

Reply:
okay.